packages/storybook/stories/Breadcrumbs.stories.jsx

```diff
@@ -29,8 +29,8 @@ const Template = args => (
 
 const defaultArgs = {};
 
-export const Default = Template.bind({});
+export const Default = Template.bind(null);
```

Comment:
I chose null based on the recommendation [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this#arrow_functions).